cmd/schedule-runner/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"cmp"
+	"context"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/donor"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/dynamo"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/event"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/localize"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/notify"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/scheduled"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/search"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/secrets"
+)
+
+func handleRunSchedule(ctx context.Context) error {
+	var (
+		eventBusName          = cmp.Or(os.Getenv("EVENT_BUS_NAME"), "default")
+		notifyBaseURL         = os.Getenv("GOVUK_NOTIFY_BASE_URL")
+		notifyIsProduction    = os.Getenv("GOVUK_NOTIFY_IS_PRODUCTION") == "1"
+		searchEndpoint        = os.Getenv("SEARCH_ENDPOINT")
+		searchIndexName       = cmp.Or(os.Getenv("SEARCH_INDEX_NAME"), "lpas")
+		searchIndexingEnabled = os.Getenv("SEARCH_INDEXING_DISABLED") != "1"
+		tableName             = os.Getenv("LPAS_TABLE")
+	)
+
+	logger := slog.New(slog.NewJSONHandler(os.Stdout, nil).
```

Comment:
it would be good to have x-ray hooked up too, but could make a new ticket for that

Reply:
Good call - I'll add a ticket to the backlog.